fe/fe-core/src/test/java/com/starrocks/common/lock/LockRandomTest.java

```diff
@@ -119,7 +119,7 @@ public void testOnlyXWithS() throws InterruptedException {
 
     @Test
     public void testOnlyXWithIS() throws InterruptedException {
-        int runTimes = 10000;
+        int runTimes = 1000;
```

Comment:
Actually I don't think it makes sense to create 1000 threads to run concurrently because there are only 8/32 CPUs.
It will be better that create 32 threads and each of them run multiple times.